compiler/src/iree/compiler/Codegen/LLVMGPU/Passes.cpp

```diff
@@ -410,8 +410,11 @@ LogicalResult isAtBoundary(Operation *op) {
 void addGPUTileAndFusePassPipeline(OpPassManager &funcPassManager,
                                    const GPUPipelineOptions &pipelineOptions,
                                    bool forROCDL) {
-  if (pipelineOptions.useIgemmConvolution) {
+  if (pipelineOptions.useIgemmConvolution ||
+      pipelineOptions.useDirectConvolution) {
```

Comment:
I think we can avoid adding this option all together, we want to limit adding new options if possible, we can run the createGPUPadConvsPass outside of any if conditions as if there is no padding_conv attribute it wont do anything.